bitnami/jasperreports/templates/deployment.yaml

```diff
@@ -204,3 +204,6 @@ spec:
         {{- if .Values.extraVolumes }}
         {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumes "context" $) | nindent 8 }}
         {{- end }}
+      {{- if .Values.dnsConfig }}
+      dnsConfig: {{- include "common.tplvalues.render" (dict "value" .Values.dnsConfig "context" $) | nindent 8 }}
+      {{- end }}
```

Comment:
Instead of adding this at the bottom, could you please move it under `spec:`, line 50 right after `tolerations` for example?

Reply:
> Instead of adding this at the bottom, could you please move it under `spec:`, line 50 right after `tolerations` for example?

Done.